frontend/ui/lib/services/places_services_html.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:async';
+import 'dart:html';
```

Comment:
Expanding the surface area of conditional imports is the opposite of what we want to do. Please consider whether there are alternative ways to do this. If the Places API has a hard dependency on `dart:html`, consider wrapping the minimal kernel with an agnostic interface that can be stubbed. Then only the _real_ implementation needs to be conditionally imported.

Note: it's possible the that the is the only way to make things work, but please confirm that if so.

Reply:
so looking at this, it's pretty gross but the underlying JS api actually seems to require the html element... https://developers.google.com/maps/documentation/javascript/places#place_details_requests

Reply:
This StackOverflow answer actually explains why this seems to be required: https://stackoverflow.com/a/30660844.

Note that it _doesn't_ have to be an actual map element, but we do need to pass some sort of visible element.